### ISSUE TITLE
[Feature request] When entering items in todo-list, the key "enter" takes you to a new item 

### ISSUE THREAD

**Issue by yazeed44** [PhieF/CarnetDocumentation#69]
It's not convenient when you have to press "Add item" every time you want to make a new item

**Comment by hurricanejosh** [comment]
yes please. also bulk import- when pasting  multi-line text, a new checkbox item is created for each line

**Comment by StyXman** [comment]
@yazeed44 at least on the NextCloud interface, Ctrl-Enter gives a new entry to fill up. And like @hurricanejosh , I think a text markup language for exporting/importing would be nice. Maybe something based on [GH's own Markdown](https://guides.github.com/features/mastering-markdown/)?
